server/tlsconfig.go

```diff
@@ -0,0 +1,110 @@
+// Copyright 2017 Pilosa Corp.
```

Comment:
should this have the other copyright/license info in addition to or in place of ours?

Reply:
Yeah good call, better to be more explicit. Updated.